xrpl/asyncio/clients/async_websocket_client.py

```diff
@@ -56,61 +56,152 @@ class AsyncWebsocketClient(AsyncClient, WebsocketBase):
     using the ``asyncio`` library to listen to incoming
     messages and do something with them, but the above will
     work fine if you want to listen indefinitely. This is how
-    you can use a Task to listen to messages without blocking::
+    you can use a Task to listen to messages without blocking. Note this
+    example can be copied directly into a file and run with no changes::
 
         import asyncio
 
         from xrpl.asyncio.clients import AsyncWebsocketClient
         from xrpl.models import Subscribe, Unsubscribe, StreamParameter
 
-        URL = "wss://some-url-to-connect-to"
+        URL = "wss://s.devnet.rippletest.net:51233"
```

Comment:
nit: I'd recommend using testnet instead of devnet here since most people aren't testing new amendments - wss://s.altnet.rippletest.net:51233/